testdata/scripts/node/validate/invalid-duplicates.txtar

```diff
@@ -0,0 +1,94 @@
+! exec chainlink node -c config.toml -s secrets.toml validate
+cmp stderr err.txt
+cmp stdout out.txt
+
+-- config.toml --
+Log.Level = 'debug'
+
+[[EVM]]
+ChainID = '1'
+
+[[EVM]]
+ChainID = '1'
+
+[[EVM.Nodes]]
+Name = 'fake'
+WSURL = 'wss://foo.bar/ws'
+HTTPURL = 'https://foo.bar'
+
+[[EVM.Nodes]]
+Name = 'fake'
+WSURL = 'wss://foo.bar/ws'
+HTTPURL = 'https://foo.bar'
+
+[[Cosmos]]
+ChainID = 'Malaga-420'
+
+[[Cosmos]]
+ChainID = 'Malaga-420'
+
+[[Cosmos.Nodes]]
+Name = 'primary'
+TendermintURL = 'http://tender.mint'
+
+[[Cosmos.Nodes]]
+Name = 'primary'
+TendermintURL = 'http://tender.mint'
+
+[[Solana]]
+ChainID = 'mainnet'
+
+[[Solana]]
+ChainID = 'mainnet'
+
+[[Solana.Nodes]]
+Name = 'primary'
+URL = 'http://solana.web'
+
+[[Solana.Nodes]]
+Name = 'primary'
+URL = 'http://solana.web'
+
+[[Starknet]]
+ChainID = 'foobar'
+
+[[Starknet]]
+ChainID = 'foobar'
+
+[[Starknet.Nodes]]
+Name = 'primary'
+URL = 'http://stark.node'
+
+[[Starknet.Nodes]]
+Name = 'primary'
+URL = 'http://stark.node'
+
+
+-- secrets.toml --
+[Database]
+URL = 'postgresql://user:pass@localhost:5432/dbname?sslmode=disable'
+BackupURL = ''
+
+[Password]
+Keystore = ''
+
+-- out.txt --
+-- err.txt --
+Error running app: detected breaking duplicates: 4 errors:
```

Comment:
nit/ I wonder if 'breaking' will be clear to node ops. The normal validation message is:
`Invalid configuration: invalid secrets: 2 errors:...`
So maybe something simple like:
```suggestion
Error running app: invalid configuration: 4 errors:
```